Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "vmm-vcpu"
+version = "0.1.0"
+authors = ["Jenny Mankin <jenny.mankin@crowdstrike.com>", 
+           "Timo Kreuzer <timo.kreuzer@reactos.org>",
+           "Alessandro Pilotti <apilotti@cloudbasesolutions.com>"]
+description = "Hypervisor-agnostic abstraction crate over virtual CPU functionality"
+repository = "https://github.com/rust-vmm/vmm-vcpu"
+readme = "README.md"
+license = "Apache-2.0 or MIT"
+           
+[dependencies]
+byteorder = "*"
+
+[target.'cfg(unix)'.dependencies]
+kvm-bindings = "0.1"
+
+[target.'cfg(unix)'.dev-dependencies]
+kvm-ioctls = "0.1.0"
```

Comment:
Just noticed that. We will have a problem here because rust doesn't allow cyclic dependencies between crates. Isn't the idea to have `kvm-ioctls` import `vmm-vcpu` as well?

Reply:
Good point about the circular dependency. This dependency isn't strictly necessary; it is only needed for the CpuId structure doc comment examples, which are run as tests when running `cargo test` (thus `dev-dependencies` being a dependency for tests only).

Those tests/examples will become outdated anyway with the upcoming crate for "KvmVec"-like generic abstractions (and CpuId and MsrEntries implementations), [ie here](https://github.com/rust-vmm/kvm-ioctls/pull/6). But when that change is incorporated, the same problem will remain: What is the best way to handle platform-specific doc comment examples in abstraction crates. This looks to be a common problem with a solution that the (rust developers agree should be allowed)[https://github.com/rust-lang/cargo/issues/4242]. But seeing as that issue is still open, I see three options, none perfect: 

1) Demonstrate API usage with specific (KVM and/or WHP) examples, which is the most instructive and useful way to show how to use the APIs, but results in this circular crate dependency when running tests. (Seems from the link above that a workaround could be to comment out `dev-dependencies` when publishing?)
2) Examples could probably be made more generic, albeit less instructive, in order to demonstrate usage without dependencies.
3) The platform-specific examples could remain, but the doc comment examples can be annotated with the `ignore` directive so that they are visible as documentation, but not actually run with `cargo test`.